dist/plugins/DefaultPlugin/resources/analysisTools/roddyTools/wrapInScript.sh

```diff
@@ -79,10 +97,7 @@ else
 
   # Set LD_LIBRARY_PATH to LD_LIB_PATH, if the script was called recursively.
   [[ ${LD_LIB_PATH-false} != false ]] && export LD_LIBRARY_PATH=$LD_LIB_PATH
-  [[ ${debugWrapInScript-false} == true ]] && set -xv
```

Comment:
As already said, these variables are only availbe, if you sourced in CONFIG_FILE or if you set it via the environment.

Reply:
In the newest version, I let BE (as quick fix) write `debugWrapInScript` via the command line parameters. Therefore it is directly available to the `wrapInScript.sh`. After that, first the CONFIG_FILE and then the job-specific parameters from the PARAMETER_FILE are read (with optional debugging turned on).